include/kimera_vio_ros/RosOnlineDataProvider.hpp

```diff
@@ -53,23 +45,23 @@ class RosOnlineDataProvider : public RosDataProviderInterface {
 
  private:
   // Left camera callback
-  void callbackStereoImages(const sensor_msgs::ImageConstPtr& left_msg,
-                            const sensor_msgs::ImageConstPtr& right_msg);
+  void callbackStereoImages(const sensor_msgs::msg::ImageConstPtr& left_msg,
+                            const sensor_msgs::msg::ImageConstPtr& right_msg);
```

Comment:
Why are all these changes in namespace necessary?

Reply:
Again, API changes for ROS2, given the extended IDL support for generated C++ interfaces:

https://index.ros.org/doc/ros2/Contributing/Migration-Guide/#update-source-code